### ISSUE TITLE
Error message not taken into account for the Agree field

### ISSUE THREAD

**Issue by benjamindavid** [verbb/formie#140]
**Description**
I'm setting the checkbox "Agree" field as required and I'm giving it a custom message. When the form is submitted with this checkbox left unchecked, it shows a generic `This field is required.` instead of the custom error message.

**Steps to reproduce**
1. Create a new form
2. Add an "Agree" field, set it as required and give it a custom error
3. Leave the Agree box unchecked and submit the form
4. The generic error shows up instead of the custom error

### Form settings
Please pick from the options below.

- Multi-page form: No
- Submission Method: Page Reload
- Client-side Validation: Yes
- Custom Form Templates: Yes

**Additional info**
- Plugin version: 1.2.17
- Craft version: 3.4.28.1
- Multi-site: No

**Comment by benjamindavid** [comment]
It looks like radio and date fields behave the same way.

Is there a way to at least translate them? I tried with static message translation but unfortunately that didn't work.

**Comment by engram-design** [comment]
I'll check this out, but static translations should certainly work for instance, `translations/en-AU`:

```
<?php

return [
    'This field is required.' => 'This field is translated yo.',
];
```

**Comment by engram-design** [comment]
Fixed in [1.2.18](https://github.com/verbb/formie/releases/tag/1.2.18)

**Comment by benjamindavid** [comment]
That was fast! Thanks @engram-design!
